pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -13,6 +14,34 @@ The
 %a{href:"https://hourofcode.com"} Hour of Code
 is a global movement reaching tens of millions of students in 180+ countries. Anyone, anywhere can organize an Hour of Code event or try any of the over 200+ one-hour tutorials, available in over 45 languages. Teachers, to view lesson plans and additional teacher resources please select your desired Hour of Code tutorial from the side menu to the left.
 
+#resources.responsive-body
+  .col-50
+    .tutorial-tile
```

Comment:
Just noting that this is another example of a component that exists in duplicate - one in haml and one in React.  Long term it'd be nice to strategize how we can have maximum re-usability across the site.

Reply:
I'd like to re-use these on the "challenge" page as well.

Reply:
The css for this is already extracted. As long as you use the same classnames and import the css file, you should have no problem re-using it.

Reply:
See https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/educate/applab.haml#L24

Reply:
Would be good to extract this as a Haml partial.  Otherwise each time we update the description text we'll also have to update `n` other places 😕.

Reply:
ohh I see - you want the content as well as the layout 👍